win_installer.py

```diff
@@ -253,6 +253,10 @@ def run(bitness, repo_root):
         print('Installing mu with', venv_python)
         subprocess.run([venv_python, '-m', 'pip', 'install', repo_root])
 
+        print('Updating pip in the virtual environment', venv_python)
+        subprocess.run(
+            [venv_python, '-m', 'pip', 'install', '--upgrade', 'pip'])
```

Comment:
For formatting consistency, I'd put the closing `)` on the next line, vertically left-aligned with `subprocess.run(`, as in, say the `shutil.copy` call a few lines down. What do you say? :)

Reply:
I'll make this minor change and then merge. :-)